flight_manager.py

```diff
@@ -100,6 +102,10 @@ def run_threads(self, sim: bool) -> None:
         logging_process = init_logger(log_queue)
         logging_process.start()
 
+        # connect to the drone
+        drone: System = System()
+        drone.connect(system_address="udp://:14540")
+
```

Comment:
This stuff isn't necessary, since the init_flight() fuction handles the case of a simulator or live drone initialization.